app/src/main/java/org/mozilla/fenix/home/recentvisits/controller/RecentVisitsController.kt

```diff
@@ -80,19 +88,30 @@ class DefaultHistoryMetadataController(
         )
     }
 
-    override fun handleRemoveRecentHistoryGroup(searchTerm: String) {
+    /**
+     * Removes a [RecentHistoryGroup] with the given title from the homescreen.
+     *
+     * @param groupTitle The title of the [RecentHistoryGroup] to be removed.
+     */
+    override fun handleRemoveRecentHistoryGroup(groupTitle: String) {
```

Comment:
I would probably keep this as `searchTerm` instead of `groupTitle` since all its subsequent callers use `searchTerm`

Reply:
But the [RecentHistoryGroup] doesn't have a `searchTerm` property and this method is called for that type.
I could rename it's `title` property to `searchTerm` but feel like `title` is more appropriate for where and how it's used. (The other item - `RecentHistoryHighlight` also has a `title` property)